core/src/config/Networks.cpp

```diff
@@ -43,6 +43,7 @@ namespace ledger {
                     {0x04, 0x88, 0xB2, 0x1E},
                     BitcoinLikeFeePolicy::PER_BYTE,
                     5420,
+                    BitcoinLikeDustPolicy::FIXED,
```

Comment:
TODO: check where this change in struct is propagated through JNI

Reply:
It's safe

Reply:
https://github.com/ledgerhq/lib-ledger-core/blob/e4d8111f264d84970d5d30e9471b1643e0d64c07/core/src/jni/jni/BitcoinLikeNetworkParameters.cpp#L16-L27